src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -59,3 +60,23 @@ export interface IncorporationFilingIF {
     }
   }
 }
+
+/** Interface for incorporation filing data saved to the Legal API. */
+export interface DissolutionFilingIF {
+  filing: {
+    header: {
+      name: string
+      certifiedBy: string
+      date: string
+      effectiveDate?: string // Optional and should be set only for future effective filings
+      filingId?: number // Optional as this is not required when building a filing - causes an error for new filings
+      folioNumber?: string // Optional to the user and only displayed for certain account types
+      isFutureEffective: boolean
+    },
+    business: {
+      legalType: CorpTypeCd,
+      identifier: string
+    },
+    dissolution: {}
```

Comment:
If this works, great! If not, you can use `any` and a comment indicating this is a untyped object. Or something like that.

Reply:
It works, and is going to be built on as soon as we being developing the filing itself. 
I'm not too worried about this at the moment :)